x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/create_analytics_form/messages.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, FC } from 'react';
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+
+import { FormMessage } from '../../hooks/use_create_analytics_form/state'; // State
+
+interface Props {
+  messages: any; // TODO: fix --> something like State['requestMessages'];
+}
+
+export const Messages: FC<Props> = ({ messages }) =>
```

Comment:
Nit: Suggest to give this a more specific name, e.g. `FormValidationMessages` / `form_validation_messages.tsx`.